pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -164,6 +165,13 @@ private static HttpUriRequest getSendSegmentUriRequest(URI uri, String downloadU
     return requestBuilder.build();
   }
 
+  private static HttpUriRequest getSegmentCompletionUriRequest(String uri, int socketTimeoutMs) {
```

Comment:
If possible, can we keep uri as type URI for consistency?

Reply:
I had it like that before, but changed it to string, so that the string-to-uri conversion is in one place.  i would like to keep it that way.